app/views/teams/show.html.erb

```diff
@@ -1,20 +1,34 @@
-<p id="notice"><%= notice %></p>
+<div class="content-wrapper">
 
-<div>
-    <%= Team.human_attribute_name("name") %><br />
-    <%= @team.name %>
-</div>
 
-<div>
-    <%= Team.human_attribute_name("users") %><br />
-    <% @team_users.each do |user| %>
-        <p>
-            <%= user.email %>
-            <%= user.users_teams.where('team_id = ?', @team.id)[0].user_team_role %>
-        </p>
-    <% end %>
+  <div class="content-box">
+    <div class="row" style="justify-content: space-between;">
+      <div class="box-header" style="flex-basis: 0; flex-grow: 1">
```

Comment:
ungünstig style hard in html schreiben, dann dafür eine klasse erstellen, besonders kommt dieser style auch bei den anderen Team views vor. Hier ggf. refactorn